### ISSUE TITLE
Asterisks displayed when widown not removed

### ISSUE THREAD

**Issue by lodumont** [gucci-on-fleek/lua-widow-control#32]
### Issue

Still using the memoir class with lua-widow-control v. 2.0.4, three asterisks are displayed on the line where the package failed to removed a widow (see below). I didn't find anything in the manual concerning this behaviour. Is there anyway to avoid it?

<img width="513" alt="Capture d’écran 2022-04-11 à 12 31 10" src="https://user-images.githubusercontent.com/61599071/162722308-f83371f3-d993-4b7c-ab8f-628d4aa597df.png">

### Expected Behavior

I wish no asterisks were displayed where widow were not removed.

### Format

LaTeX

### Distribution

TeX Live

### Distribution Version

TeX Live 2022

### Lua-widow-control Version

2.0.4

**Comment by gucci-on-fleek** [comment]
That's definitely not supposed to happen. It's not in the manual because it's a bug. 

I'm actually not entirely sure what's causing this. Please compiling the document with
```latex
\usepackage[debug=true]{lua-widow-control}
```
then upload the log file in a reply (it'll probably be fairly large). I have a few guesses as to what's going on, but I've never seen this happen before so it's hard to be sure without a log file.

**Comment by lodumont** [comment]
Here is the (indeed pretty big) log file:

[lua-widow-control-debug.log](https://github.com/gucci-on-fleek/lua-widow-control/files/8463680/lua-widow-control-debug.log)

Here are the lines around p. 225:

```
LWC:                   iflwc [224<../Images//Typo_coupelle_Bader91.jpg>]
<../Images//Typo_coupelle_K91.jpg, id=4755, 446.8695pt x 284.7438pt>
File: ../Images//Typo_coupelle_K91.jpg Graphic file (type jpg)
<use ../Images//Typo_coupelle_K91.jpg>
Package luatex.def Info: ../Images//Typo_coupelle_K91.jpg  used on input line 8
59.
(luatex.def)             Requested size: 426.0pt x 271.45659pt.
LWC (outputpenalty  ): -10004 0
LWC (outputpenalty  ): -10003 0
LWC (outputpenalty  ): -51 0
Underfull \vbox (badness 10000) has occurred while \output is active []

LWC:                   iflwc
LWC:                   iflwc [225]
LWC (outputpenalty  ): -51 0
Underfull \vbox (badness 1127) has occurred while \output is active []

LWC:                   iflwc
LWC:                   iflwc [226<../Images//Typo_coupelle_K91.jpg>]
<../Images//Typo_coupelle_Stockhammer.jpg, id=4786, 416.5161pt x 194.6472pt>
File: ../Images//Typo_coupelle_Stockhammer.jpg Graphic file (type jpg)
<use ../Images//Typo_coupelle_Stockhammer.jpg>
Package luatex.def Info: ../Images//Typo_coupelle_Stockhammer.jpg  used on inpu
t line 908.
```

**Comment by gucci-on-fleek** [comment]
I don't actually think that lua-widow-control is causing this. Due to a different bug, it looks like lua-widow-control completely disabled itself beginning after "Cette thèse de doctorat vise…" on page 14 until the end of the document. That should be fixed in 39d26b0 (not on CTAN/TeX Live yet).

---

If you remove 
```latex
\usepackage{lua-widow-control}
```
from your document, does the issue go away? 

The asterisks look fairly similar to the memoir ["Fancy Anonymous Breaks"](https://texdoc.org/serve/memoir/0#page=138), so maybe something else is inserting them? This could still be caused by lua-widow-control, but it seems unlikely.

**Comment by gucci-on-fleek** [comment]
Yep, I'm able to reproduce this. It seems to happen whenever memoir is used and an orphan is detected in a two-line paragraph.

Reproduction:
```latex
\documentclass{memoir}

\usepackage[debug=true]{lua-widow-control}

\begin{document}
    \vspace*{44\baselineskip}

    One two three four five six seven eight nine ten eleven twelve thirteen fourteen fifteen sixteen seventeen eighteen nineteen twenty twenty-one.
\end{document}
```
<image src="https://user-images.githubusercontent.com/49086429/162900414-52643231-c809-4c3c-bda2-a7437210040e.png" width=75%/>

---

I have absolutely no idea what is causing this. I'll investigate and try and fix it though.

**Comment by gucci-on-fleek** [comment]
Ok, so the memoir defines its output routine as:
```latex
\def\pfbreakOutput{%
  \ifnum\outputpenalty=\pen@ltyabovepfbreak
    \nopfbreakOutput
    \pfbre@kdispl@y
    \nobreak
    \vskip-\pfbreakskip
  \else\ifnum\outputpenalty=\pen@ltybelowpfbreak
    \unvbox 255\relax
    \nobreak
    \vskip-\pfbreakskip
    \pfbre@kdispl@y
    \break
  \else
    \nopfbreakOutput
  \fi
  \fi}
\output={\pfbreakOutput}
```

If we go back a little further in the file, we find
```latex
\newcommand*{\pen@ltyabovepfbreak}{2}
\newcommand*{\pen@ltybelowpfbreak}{-4}
```

What is happening is that lua-widow-control sets `\clubpenalty=1` and `\widowpenalty=1`. In a two-line paragraph split down the middle, there is both a widow and an orphan/club, so the `\outputpenalty` is set to `2`. Memoir happens to use an output penalty of `2` to insert asterisks. Through bad luck, these two numbers happen to collide.

---

Placing
```latex
\makeatletter
\def\pen@ltyabovepfbreak{-7}
\makeatother
```
somewhere in your preamble should fix this. I'll try and get a fix out to CTAN/TeX Live in a few days.

**Comment by lodumont** [comment]
Well that was tricky! Thank you very much for looking into it, that solved my issue.

**Comment by gucci-on-fleek** [comment]
This has been fixed in v2.0.5, which has been sent off to CTAN. It should be on TeX Live in about 2 days.
